source/_integrations/insteon.markdown

```diff
@@ -63,7 +63,7 @@ The Insteon configuration panel allows for product specific configuration of Ins
 To open the Insteon configuration panel:
 
 1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.
-2. Select the **Insteon** {% term integration %}. Then, select **Configure** to open the Insteon configuration panel.
+2. Select the **Insteon** {% term integration %}. Then, select **Add** to open the Insteon configuration panel.
```

Comment:
I think this is also related to an already existing config entry, those related to the options flow?